src/Gui/PythonEditor.cpp

```diff
@@ -1466,9 +1466,9 @@ bool PythonEditorCodeAnalyzer::afterChoiceInserted(JediBaseDefinitionObj *obj, i
         return false;
 
     QString type = obj->type();
-    bool paranthesis = false;

```

Comment:
i search all `paranthesis` and found 3 in the whole branch all in the same file so I fixed them. Hope that's OK with you.